.tekton/ostree-build.yaml

```diff
@@ -0,0 +1,506 @@
+        params:
+          - name: name
+            value: git-clone
+          - name: bundle
+            value: quay.io/redhat-appstudio-tekton-catalog/task-git-clone:0.1@sha256:d9e1ab10d72953e7a85dab69b8b96f5b41580a6d4026f77b6a5ba6f3ed227cc3
+          - name: kind
+            value: task
+        resolver: bundles
+      when:
+        - input: $(tasks.init.results.build)
+          operator: in
+          values:
+            - "true"
+      workspaces:
+        - name: output
+          workspace: workspace-arm64
+        - name: basic-auth
+          workspace: git-auth
+    - name: clone-repository-s390x
+      params:
+        - name: url
+          value: $(params.git-url)
+        - name: revision
+          value: $(params.revision)
+      runAfter:
+        - init
+      taskRef:
+        kind: Task
+        params:
+          - name: name
+            value: git-clone
+          - name: bundle
+            value: quay.io/redhat-appstudio-tekton-catalog/task-git-clone:0.1@sha256:d9e1ab10d72953e7a85dab69b8b96f5b41580a6d4026f77b6a5ba6f3ed227cc3
+          - name: kind
+            value: task
+        resolver: bundles
+      when:
+        - input: $(tasks.init.results.build)
+          operator: in
+          values:
+            - "true"
+      workspaces:
+        - name: output
+          workspace: workspace-s390x
+        - name: basic-auth
+          workspace: git-auth
+    - name: clone-repository-ppc64le
+      params:
+        - name: url
+          value: $(params.git-url)
+        - name: revision
+          value: $(params.revision)
+      runAfter:
+        - init
+      taskRef:
+        kind: Task
+        params:
+          - name: name
+            value: git-clone
+          - name: bundle
+            value: quay.io/redhat-appstudio-tekton-catalog/task-git-clone:0.1@sha256:d9e1ab10d72953e7a85dab69b8b96f5b41580a6d4026f77b6a5ba6f3ed227cc3
+          - name: kind
+            value: task
+        resolver: bundles
+      when:
+        - input: $(tasks.init.results.build)
+          operator: in
+          values:
+            - "true"
+      workspaces:
+        - name: output
+          workspace: workspace-ppc64le
+        - name: basic-auth
+          workspace: git-auth
+    - name: build-container-amd64
+      params:
+        - name: IMAGE
+          value: $(params.output-image)-amd64
+        - name: IMAGE_FILE
+          value: $(params.image-file)
+        - name: CONTEXT
+          value: $(params.path-context)
+        - name: IMAGE_EXPIRES_AFTER
+          value: $(params.image-expires-after)
+        - name: COMMIT_SHA
+          value: $(tasks.clone-repository.results.commit)
+        - name: PLATFORM
+          value: linux/amd64
+        - name: BUILDER_IMAGE
+          value: quay.io/bcook/ostree-builder:latest
+      runAfter:
+        - clone-repository
+      taskRef:
+        params:
+          - name: url
+            value: https://github.com/stuartwdouglas/build-definitions/
```

Comment:
Ah, OK this is a pretty load bearing part...looks like this points to
https://github.com/stuartwdouglas/build-definitions/commit/bc7f072fe752b259ce54a755dabd62b48d44474b